google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateBackupRequest.java

```diff
@@ -58,6 +58,20 @@ public CreateBackupRequest setExpireTime(Instant expireTime) {
     return this;
   }
 
+  public CreateBackupRequest setBackupType(Backup.BackupType backupType) {
+    Preconditions.checkNotNull(backupType);
+    requestBuilder.getBackupBuilder().setBackupType(backupType.toProto());
+    return this;
+  }
+
+  public CreateBackupRequest setHotToStandardTime(Instant hotToStandardTime) {
```

Comment:
please document what happens at that instant? Also is this is a required field for HotBackups?